packages/core-flows/src/user/steps/update-users.ts

```diff
@@ -0,0 +1,38 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import { IUserModuleService, UpdateUserDTO } from "@medusajs/types"
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+
+export const updateUsersStepId = "update-users-step"
+export const updateUsersStep = createStep(
+  updateUsersStepId,
+  async (input: UpdateUserDTO[], { container }) => {
+    const service: IUserModuleService = container.resolve(
+      ModuleRegistrationName.USER
+    )
+
+    const originalUsers = await service.list({
+      id: input.map((u) => u.id),
```

Comment:
suggestion: For safety guard, as I have seen it occured in the modules, I would store the ids separately and then id ids.length then call the lists and the update wdyt?